### ISSUE TITLE
[ts]: Typing Template Refs

### ISSUE THREAD

**Issue by WuChenDi** [nuxt-modules/turnstile#341]
### 📚 What are you trying to do?

Hello everyone, my apologies for the interruption.

How can one determine the type of the `NuxtTurnstile` component in practical development?

```vue
import { NuxtTurnstile } from '#components'

// const turnstile = ref<InstanceType<typeof NuxtTurnstile> | null>()

const turnstile = useTemplateRef<typeof NuxtTurnstile>(null)
```
![image](https://github.com/user-attachments/assets/300223d6-74ea-48a6-87c5-016f134304ce)

![image](https://github.com/user-attachments/assets/a6177196-122d-4669-a6ee-4abaca128fa6)

![image](https://github.com/user-attachments/assets/bdbd222c-acaa-486f-b251-c076ce22cfc7)

**Comment by Yizack** [comment]
It works for me 🤔

```js
import { NuxtTurnstile } from "#components";
const turnstile = ref<InstanceType<typeof NuxtTurnstile> | null>();
```

![image](https://github.com/user-attachments/assets/44ce3964-5705-44ca-be60-ee460bededb9)

![image](https://github.com/user-attachments/assets/fdceebf4-24bb-4a3d-8533-7e5d08ab4a14)

**Comment by WuChenDi** [comment]
@Yizack Thank you for your response.

As mentioned in my initial reply, I attempted to write the code in this manner; however, the result was `any`, which is of no use to me.

```yaml
    devDependencies:
      '@nuxtjs/turnstile':
        specifier: ^0.9.7
        
```

![image](https://github.com/user-attachments/assets/cdabe0b0-f92b-43ed-8bc5-524fbf57ab26)

**Comment by Yizack** [comment]
I see, have you tried re-generating the types again using `npx nuxi prepare` I believe sometimes when reopening the project it gives `any`, you could always use as workaround if you need it

```js
const turnstile = ref<{ reset: () => void }>();
```

**Comment by WuChenDi** [comment]
No, I have attempted to execute `npx nuxi prepare`, but regrettably, the outcome remains `any`. 

Currently, this is how I have approached the issue.

 `ref<{ reset: () => void }>();`

**Comment by dargmuesli** [comment]
Closing due to missing minimal reproduction. If submitted, I'll reopen.
